library-api/src/main/java/au/csiro/pathling/library/PathlingContext.java

```diff
@@ -315,4 +360,75 @@ public Dataset<Row> encodeBundle(@Nonnull final Dataset<Row> stringBundlesDF,
     return encodeBundle(stringBundlesDF, resourceName, FhirMimeTypes.FHIR_JSON);
   }
 
+  @Nonnull
+  public Dataset<Row> memberOf(@Nonnull final Dataset<Row> dataset,
+      @Nonnull final Column coding, @Nonnull final String valueSetUri,
+      @Nonnull final String outputColumnName) {
+
+    final Column codingArrayCol = when(coding.isNotNull(), array(coding))
```

Comment:
I think it would be useful to refactor this functionality so that it can also be reused in MemebrOf function.

Maybe something like 

``` 
class TerminologyFunctions {

    private final TerminologyServiceFactory terminologyServiceFactory;

    @Nonnull
    public Dataset<Row> memberOf(@Nonnull final Dataset<Row> dataset,
      @Nonnull final Column coding, @Nonnull final String valueSetUri,
      @Nonnull final String outputColumnName);
      
      // other terminology functions.```

Reply:
Resolved this for memberOf, translate and subsumes in: d8e074150993d4721bd6f95c82d6db9cf946e44f, dccf4bfa3198e040cb3a168d80f413582ad014df and 8b3df9369113fd5a3cb1b314862d99e7e9df226a